src/src/AdminBundle/Resources/views/modal.html.twig

```diff
@@ -0,0 +1,22 @@
+<div class="modal fade" id="{{ modal_id }}" tabindex="-1" role="dialog" aria-labelledby="deleteModalLabel"
```

Comment:
tuto ma byt `id="deleteModal{{ modal_id }}"` namiesto `id="{{ modal_id }}"`
Presvedc sa prosim ci to funguje. ak nie, oprav to prosim.

Reply:
vsetko funguje, to som si preklikal. ak si pozries tak ako `modal_id` posielam (napr.) `modal_id: 'deleteModal' ~ admin.id` takze v konecnom dosledku je to spojene. + tu mi islo o to, aby sa link vdaka ktoremu sa modal zobrazuje, renderoval na rovnakom mieste ako sa nastavuje ID pre modal